desktop/install/linux-install.md

```diff
@@ -268,6 +270,48 @@ Docker Desktop for Linux runs a Virtual Machine (VM) for the following reasons:
 
     As such, we have adjusted the default memory available to the VM in DD4L. You can tweak this setting to your specific needs by using the **Memory** slider within the **Settings** > **Resources** tab of Docker Desktop.
 
+## File sharing
+
+Docker Desktop for Linux uses [virtiofs](https://virtio-fs.gitlab.io/) as the
+default (and currently only) mechanism to enable file sharing between the host
+and Docker Desktop VM. In order not to require elevated privileges, without
+unnecessarily restricting operations on the shared files, Docker Desktop runs
+the file sharing service (`virtiofsd`) inside a user namespace (see
+`user_namespaces(7)`) with UID and GID mapping configured. As a result Docker
+Desktop relies on the host being configured to enable the current user to use
+subordinate ID delegation. For this to be true `/etc/subuid` (see `subuid(5)`)
+and `/etc/subgid` (see `subgid(5)`) have to be present. Docker Desktop only
+supports subordinate ID delegation configured via files. In case of both,
+`/etc/subuid` and `/etc/subgid`, Docker Desktop uses the first entry
+corresponding to the current user, to set up mappings for IDs above 0, and map
+the current users ID and GID to 0.
+
+| ID in container | ID on host                                                                       |
+| --------------- | -------------------------------------------------------------------------------- |
+| 0 (root)        | ID of the user running DD (e.g. 1000)                                            |
+| 1               | 0 + beginning of ID range specified in `/etc/subuid`/`/etc/subgid` (e.g. 100000) |
+| 2               | 1 + beginning of ID range specified in `/etc/subuid`/`/etc/subgid` (e.g. 100001) |
+| 3               | 2 + beginning of ID range specified in `/etc/subuid`/`/etc/subgid` (e.g. 100002) |
+| ...             | ...                                                                              |
+
+If `/etc/subuid` and `/etc/subgid` are missing, they will need to be created.
+Both should contain entries in the form -
+`<username>:<start of id range>:<id range size>`. E.g. to allow a user whose
+username is `myusername` and group name is `mygroupname` to use IDs from 100000
+to 165535:
+
+```console
+$ echo 'myusername:100000:65536' | sudo tee /etc/subuid
+$ echo 'mygropname:100000:65536' | sudo tee /etc/subgid
+```
+
+In this scenario if a shared file is `chown`ed inside a Docker Desktop container
+to be owned by a user with a UID of 1000 it will show up on the host as owned by
+a user with a UID of 100999. This has the unfortunate side effect of preventing
+easy access to such a file on the host. The problem can be resolved by creating
+a group with the new GID and adding our user to it, or by setting a recursive
+ACL (see `setfacl(1)`) for folders shared with the Docker Desktop VM.
```

Comment:
ACL looks like interesting , are there filesystems where they are not supported?

Reply:
Yes, there are, and IIRC some filesystem drivers can be built with ACL support disabled. Though most full featured file systems (extX, btrfs, appfs, ...) should support ACLs by default.